scripts/install_local.py

```diff
@@ -118,14 +118,14 @@ def execute_command(args, cwd=None):
         UI_APP_PATH = UI_PATH + "/app"
         
         print("Updating UI files...")
+        execute_command(["rm", "-rf", "node_modules"], cwd=join(cwd, UI_COMMON_PATH))
```

Comment:
Hm, is this something we want to do every time? That means that running `install_local` is going to be a pretty expensive operation.

Reply:
@agiron123

Reply:
Technically correct, it will take longer. This ensures that people have a clean build and that things should work properly.

We can also add a flag to allow people to choose whether or not they want to do a clean build. Personally I'm of the camp of doing clean builds every time as there's less moving parts and things to worry about.

In the future I'd like to look into some other build tools to use. I'm pretty sure we can get much faster build times by adopting a different build tool for common.